src/main/scala/org/downtowndailybread/ApiMain.scala

```diff
@@ -1,41 +1,75 @@
 package org.downtowndailybread
 
+
 import akka.actor.ActorSystem
-import akka.http.scaladsl.server.ExceptionHandler
 import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.StatusCode
+import akka.http.scaladsl.model.StatusCodes._
+import akka.http.scaladsl.server.AuthenticationFailedRejection.CredentialsMissing
 import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.{RejectionHandler, _}
 import akka.stream.ActorMaterializer
+import ch.megard.akka.http.cors.scaladsl.CorsDirectives._
+import org.downtowndailybread.auth.Authentication
 import org.downtowndailybread.controller.Routes
-import org.downtowndailybread.model.exceptions.DDBException
-import org.downtowndailybread.model.json.JsonSupport
+import org.downtowndailybread.exceptions.{DDBException, UnauthorizedException}
+import org.downtowndailybread.json.JsonSupport
+import org.downtowndailybread.request.DatabaseSource
+import org.flywaydb.core.Flyway
 
 import scala.io.StdIn
 
-object ApiMain extends JsonSupport with Routes{
+object ApiMain extends JsonSupport with Routes with ApiGlobalResources {
   def main(args: Array[String]) {
 
+    val flyway = Flyway.configure.dataSource(DatabaseSource.ds).load()
+    flyway.migrate()
```

Comment:
A flyway migration can happen in a separate script, but it should not be done in our main.

Reply:
Didn't fix this yet, but will.